diff -up bino-1.6.6/src/media_object.cpp.omv~ bino-1.6.6/src/media_object.cpp
--- bino-1.6.6/src/media_object.cpp.omv~	2018-06-18 03:17:21.081416774 +0200
+++ bino-1.6.6/src/media_object.cpp	2018-06-18 03:18:31.934412870 +0200
@@ -905,8 +905,6 @@ void media_object::open(const std::strin
 #ifdef FF_API_LOWRES
             lowres = codec_ctx->lowres;
 #endif
-            if (lowres || (codec && (codec->capabilities & CODEC_CAP_DR1)))
-                codec_ctx->flags |= CODEC_FLAG_EMU_EDGE;
         }
         // Find and open the codec. AV_CODEC_ID_TEXT is a special case: it has no decoder since it is unencoded raw data.
         if (codec_ctx->codec_id != AV_CODEC_ID_TEXT && (!codec || (e = avcodec_open2(codec_ctx, codec, NULL)) < 0))
